org.gnupg.GPA.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.gnupg.GPA
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: gpa
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  #- --filesystem=~/.gnupg:rw
+  #- --filesystem=xdg-run/gnupg:rw
+  - --filesystem=host
+  - --filesystem=/tmp
+
+modules:
+
+  - shared-modules/gtk2/gtk2.json
+
+  - name: gpgme
+    sources:
+      - type: archive
+        url: https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.16.0.tar.bz2
+        sha256: 6c8cc4aedb10d5d4c905894ba1d850544619ee765606ac43df7405865de29ed0
+
+  - name: gpa
+    cleanup:
+      - "*.a"
+      - "*.la"
+      - "/cache"
+      - "/include"
+      - "/lib/*.a"
+      - "/lib/pkgconfig"
+      - "/man"
+      - "/share/aclocal"
+      - "/share/devhelp"
+      - "/share/gtk-doc"
+      - "/share/man"
+      - "/share/pkgconfig"
+    buildsystem: simple
+    sources:
+      - type: git
+        url: git://git.gnupg.org/gpa.git
+        commit: faaf0b7b72bd3c4ed02490132a693d0692b45bfb
+
+    build-commands:
+      - ./autogen.sh
+      - ./configure --prefix=${FLATPAK_DEST}
+      - make
+      - make install
+      - make -k check
+    post-install:
+      - install -Dm644 ${FLATPAK_DEST}/share/pixmaps/gpa.png ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/org.gnupg.GPA.png
```

Comment:
a 128x128 icon is required (or a SVG)

Also I would just copy without renaming and just `rename-icon` making the `desktop-file-edit` unecessary.

Reply:
GPA has historically supplied only [a 48x48 icon](https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gpa.git;a=blob;f=gpa.png;h=5d612356a1498ebc40816841fa8d809ffb908b62;hb=HEAD) as far as I remember, and upscaling it would make it even worse.

Choosing a 3rd party icon like [the one from the Papirus theme](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/blob/49074f0cdecbda3f463f68a8af2e95804093b44e/Papirus/64x64/apps/gpa.svg) could be a better alternative but wouldn't be true to the original project sources.